Makefile

```diff
@@ -0,0 +1,48 @@
+SHELL:=/bin/bash
```

Comment:
missing copyright text at the beginning of the file.

Reply:
Done